x-pack/test/functional/apps/maps/es_geo_grid_source.js

```diff
@@ -36,20 +37,28 @@ export default function ({ getPageObjects, getService }) {
           beforeTimestamp = await getRequestTimestamp();
         });
 
-        it('should not rerequest when zoom changes do not cause geohash precision to change', async () => {
-          await PageObjects.maps.setView(DATA_CENTER_LAT, DATA_CENTER_LON, 2);
+        it('should not rerequest when pan changes do not cause geotile_grid precision to change', async () => {
```

Comment:
This should be `should not rerequest when pan changes do not move map view area outside of buffer`